src/test/java/org/mitre/synthea/world/concepts/LossOfCareHealthRecordTest.java

```diff
@@ -90,74 +93,80 @@ public void personRunsOutOfIncomeWithNoInsurance() {
-    person.record.encounterEnd(time, EncounterType.WELLNESS);
+    person.record.encounterEnd(time, EncounterType.INPATIENT);
     // Person has enough income for one more copay.
     assertTrue(person.defaultRecord.encounters.contains(coveredEncounter1));
     assertFalse(person.lossOfCareRecord.encounters.contains(coveredEncounter1));
 
     // Second encounter is covered and copay is affordable.
-    Encounter coveredEncounter2 = person.encounterStart(time, EncounterType.WELLNESS);
+    Encounter coveredEncounter2 = person.encounterStart(time, EncounterType.INPATIENT);
     coveredEncounter2.codes.add(code);
     coveredEncounter2.provider = new Provider();
-    person.record.encounterEnd(time, EncounterType.WELLNESS);
+    person.record.encounterEnd(time, EncounterType.INPATIENT);
     // Person is in debt $1. They should switch to no insurance not recieve any further care.
     assertTrue(person.defaultRecord.encounters.contains(coveredEncounter2));
     assertFalse(person.lossOfCareRecord.encounters.contains(coveredEncounter2));
 
     // Third encounter is uncovered and unaffordable.
-    Encounter uncoveredEncounter3 = person.encounterStart(time, EncounterType.WELLNESS);
+    Encounter uncoveredEncounter3 = person.encounterStart(time, EncounterType.INPATIENT);
     uncoveredEncounter3.codes.add(code);
     uncoveredEncounter3.provider = new Provider();
-    person.record.encounterEnd(time, EncounterType.WELLNESS);
+    person.record.encounterEnd(time, EncounterType.INPATIENT);
     // Person should have this record in the uncoveredHealthRecord.
     assertFalse(person.defaultRecord.encounters.contains(uncoveredEncounter3));
     assertTrue(person.lossOfCareRecord.encounters.contains(uncoveredEncounter3));
     // Person should now have no insurance.
-    assertTrue(person.coverage.getPayerAtTime(time).equals(Payer.noInsurance));
+    assertTrue(person.coverage.getPlanAtTime(time)
+        .equals(PayerManager.getNoInsurancePlan()));
   }
 
   @Test
   public void personRunsOutOfIncomeDueToMonthlyPremium() {
-
     Person person = new Person(0L);
     person.attributes.put(Person.BIRTHDATE, time);
     person.attributes.put(Person.GENDER, "F");
-    person.coverage.setPayerAtTime(time, testPrivatePayer);
+    InsurancePlan plan = testPrivatePayer.getPlans().iterator().next();
+    person.coverage.setPlanAtTime(time, plan);
     person.setProvider(EncounterType.WELLNESS, new Provider());
     Code code = new Code("SNOMED-CT","705129","Fake Code");
-    // Set person's income to be $1 lower than the cost of 8 monthly premiums.
-    person.attributes.put(Person.INCOME, (int) (testPrivatePayer.getMonthlyPremium() * 8) - 1);
+    // Set person's income to be $1 lower than the cost of 7 monthly premiums.
+    person.attributes.put(Person.INCOME, (int) (plan.getMonthlyPremium() * 7) - 1);
 
     // Pay monthly premium 8 times.
-    long oneMonth = Utilities.convertTime("years", 1) / 12;
+    long oneMonth = Utilities.convertTime("months", 1);
```

Comment:
I vaguely recall this being written with `years / 12` because there is no standard "month" length (28 days to 31 days) and dividing a year by 12 got you closer on the time over many (e.g., 8) months.

Utilities assumes a month of exactly 30 days.

As long as the test still works fine, but over 7 or 8 months this will be 3 to 4 days of difference.

Reply:
Oh, this probably also explains why you had to change the income to 7 months instead of 8.